ZeeguuAPI/ZeeguuAPITests/ZeeguuAPITests.swift

```diff
@@ -47,6 +65,12 @@ class ZeeguuAPITests: XCTestCase {
 		testLock.wait()
 	}
 	
+	func testZZZLogout() {
```

Comment:
Ah. I didn't realize the alphabeticallness of the test ordering. I remember that the order as shown in the left panel looked quite random, but that's because that order is not alphabetical :) 

I guess the alternative of calling login/logout in the setUp tearDown would be less efficient... so this is cool!